package TryCatch;

public class Example3 {



        static void handlingException() throws Exception{

//            try {

                throw new Exception();

//            } catch (NumberFormatException e) {

//                System.out.println("호출된 메소드에서 예외가 처리됨!");
//
//            }

        }
